index.bs

```diff
@@ -304,9 +304,15 @@ partial interface NeuralNetworkContext {
 ## Model ## {#api-model}
 <script type=idl>
 enum CompilationPreference {
+  // No preference. Let the user agent decide the type of compilation most 
+  // suitable and broadly available. This is the default option.
+  "default",
```

Comment:
Do you think whether we can make the compilation preference as an optional argument? Like
```webidl
interface Model {
  Promise<Compilation> createCompilation(optional CompilationPreference preference);
};
```

Similar to WebGPU's [`requestAdapter`](https://gpuweb.github.io/gpuweb/#gpu-interface).

The "no preference" can be fulfilled by calling `createCompilation()`. With that, the example code would look like
```
const compilation = await model.createCompilation('fast-answer');
if (!compilation) {
  compilation = await model.createCompilation();
}
```

Reply:
I like the optional arg idea. Thanks.